modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/types/src/main/resources/META-INF/resources/Checkbox/Checkbox.d.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React from 'react';
+declare const Main: React.FC<IProps>;
+interface IProps extends ICheckboxProps {
+	predefinedValue?: boolean | String[];
+	readOnly?: boolean;
+	showAsSwitcher?: boolean;
+	showMaximumRepetitionsInfo?: boolean;
+	systemSettingsURL: string;
+	value?: boolean;
+	visible?: boolean;
+}
+interface ICheckboxProps {
+	checked: boolean;
+	disabled?: boolean;
+	label?: string;
+	name: string;
+	onChange: (event: {
+		target: {
+			value: boolean;
+		};
+	}) => void;
+	required?: boolean;
+	showLabel?: boolean;
+}
```

Comment:
Is this file up to date? I'm asking that because I wasn't able to find the `ISwitcherProps` interface declaration

Reply:
This is strange, because it is in TS file and I ran the `yarn build` before send this changes.

Reply:
I rerun now and don't had changes caused by `yarn build`.